src/Core/Managed/Shared/Extensibility/Metrics/Metric.cs

```diff
@@ -1,13 +1,12 @@
-﻿namespace Microsoft.ApplicationInsights.Extensibility
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Metrics
```

Comment:
Well, but isn't it the purpose of a Beta to be able to iterate breaking changes for the new Beta features?